website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -370,6 +370,12 @@ components: sources: internal_metrics: {
 			default_namespace: "vector"
 			tags:              component_received_events_total.tags
 		}
+		component_received_bytes: {
```

Comment:
Ah, actually this needs to be added to each of the source cue files too like here: https://github.com/vectordotdev/vector/blob/59b3d7467777992e94c193d4e32321b179d20b22/website/cue/reference/components/sources/socket.cue#L113-L117